APPEND_ONLY flag + prevent incrby/decrby ooo (#451)
RedisTimeSeries/RedisTimeSeries: docs/commands.md

Append (or create and append) a new sample to the series.

```sql
TS.ADD key timestamp value [APPEND_ONLY] [RETENTION retentionTime] [UNCOMPRESSED] [LABELS label value..]
```

* timestamp - UNIX timestamp of the sample. `*` can be used for automatic timestamp (using the system clock)
* value - numeric data value of the sample (double)

> * APPEND_ONLY (optional) - Prevent back-filling of out-of-order samples

Reviewer: ```suggestion
* APPEND_ONLY (optional) - Only allow to insert samples in order.
```

Append new samples to a list of series.

```sql

> TS.MADD [APPEND_ONLY] key timestamp value [key timestamp value ...]

Reviewer 2: @ashtul I don't think we need it on MADD, we don't have the other attributes either like UNCOMPRESSED

Reviewer: @gkorland IIUC because we don't have any flags in `MADD` adding one would be weird. on the other hand not adding it leads to other incompatible API.

Author: @gkorland @danni-m 
The UNCOMPRESSED option is for the `create` option in TS.ADD which isn't available for TS.MADD.
I agree it is a bit weird but otherwise, there is no option to prevent OOO

Reviewer 2: It can also create weird parsing issues in case the user key is `APPEND_ONLY`, I would have avoid it on MADD

### TS.INCRBY/TS.DECRBY

> Creates a new sample that increments/decrements the latest sample's value. Append only support.

Reviewer: ```suggestion
Creates a new sample that increments/decrements the latest sample's value. 

> Note: These commands are supported only in append only mode.
```
